src/logic/team.ts

```diff
@@ -156,42 +211,66 @@ export const members: Array<{name: string; role: string; avatar: string; twitter
 
   {
     name: 'Abhinaba Chowdhury',
-    role: 'Events Lead of CodeChef Chapter',
+    role: 'Veteran, Events Lead of CodeChef Chapter 2021-22',
     avatar: 'https://avatars.githubusercontent.com/u/59314943?v=4',
     twitter: '',
     linkedin: 'https://www.linkedin.com/in/abhichowdhury7721',
     github: 'https://github.com/AbhiC7721',
   },
   {
     name: 'Manish Kundu',
-    role: 'CP Team ',
+    role: 'Veteran, CP Team 2021-22',
     avatar: 'https://avatars.githubusercontent.com/u/35730135?v=4',
     twitter: 'https://twitter.com/mkmanish17',
     linkedin: 'https://www.linkedin.com/in/manishkundu17',
     github: 'https://github.com/iammanish17',
   },
   {
     name: 'Priyanshi Agarwal',
-    role: 'CP Team',
-    avatar: 'https://pbs.twimg.com/profile_images/1363153542991646726/TTqwWoYQ_400x400.jpg',
+    role: 'Veteran, CP Team 2021-22',
+    avatar: 'https://media.licdn.com/dms/image/C4E03AQHehCzod_JKyQ/profile-displayphoto-shrink_400_400/0/1624336510143?e=1677110400&v=beta&t=YJB0K9FORlCJ0xAQOqqbrEQlwoSNIucnVXA-ePbRRQw',
     twitter: 'https://twitter.com/priyanshi3008',
     linkedin: 'https://www.linkedin.com/in/priyanshi-agarwal-3008/',
     github: 'https://github.com/priyanshi-3008',
   },
-  {
-    name: 'Neeraj Pratap Hazarika',
-    role: 'Video Editor',
-    avatar: 'https://avatars.githubusercontent.com/u/72177954?s=60&v=4',
-    twitter: '',
-    linkedin: 'https://www.linkedin.com/in/neeraj-pratap-hazarika-8800301b8/',
-    github: 'https://github.com/NeerajHazarika',
-  },
   {
     name: 'Devanshu Chatterjee',
-    role: 'Video Editor',
-    avatar: 'https://avatars.githubusercontent.com/u/78012571?s=400&u=b497b61c1dcdd0ed5b2b5c0f602bcef4698ee097&v=4',
+    role: 'Veteran, Video Editor 2021-22',
+    avatar: 'https://media.licdn.com/dms/image/C4D03AQGt5zAeGIX-wQ/profile-displayphoto-shrink_400_400/0/1653216699224?e=1677110400&v=beta&t=oIc10O7XA3XXsYQff9R3THYxssYJnyglKFRHkAEI0Jk',
     twitter: '',
     linkedin: 'https://www.linkedin.com/in/devanshu-chatterjee-517645200/',
     github: 'https://github.com/01-dc',
   },
+  {
```

Comment:
can move our seniors to alumni

Reply:
moved [👍](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjRpJ7khoj8AhWyoekKHa09B4MQFnoECAoQAQ&url=https%3A%2F%2Femojipedia.org%2Fthumbs-up%2F&usg=AOvVaw3qbAeyzOrK0PiArZoxYZcj)

Reply:
i meant there's a separate alumni page (in alumni.ts) you can update them there according to year